addons/l10n_se/data/account.account.template.csv

```diff
@@ -252,7 +243,6 @@
 "a6550","6550","Konsultarvoden","account.data_account_type_expenses","l10nse_chart_template","False"
 "a6560","6560","Serviceavgifter till branschorganisationer","account.data_account_type_expenses","l10nse_chart_template","False"
 "a6570","6570","Bankkostnader","account.data_account_type_expenses","l10nse_chart_template","False"
-"a6580","6580","Advokat- och rättegångskostnader","account.data_account_type_expenses","l10nse_chart_template","False"
```

Comment:
This is not listed as the recommended minimum. However, it's included in the complete BAS 2020 and should not be removed.

Reply:
Not in minimum ( https://github.com/odoo/odoo/pull/53092#discussion_r454259845 )